src/app/loadout/plug-drawer/PlugDrawer.tsx

```diff
@@ -77,8 +69,10 @@ export default function PlugDrawer({
   const defs = useD2Definitions()!;
   const language = useSelector(languageSelector);
   const [query, setQuery] = useState(initialQuery || '');
-  const [selected, setSelected] = useState(() =>
-    assignPlugsToPlugSets(plugSets, initiallySelected)
+  const [internalPlugSets, setInternalPlugSets] = useState(() =>
+    plugSets
+      .map((plugSet) => ({ ...plugSet, plugs: plugSet.plugs.sort(sortPlugs) }))
```

Comment:
Yeah it would, it won't break anything but you are right I shouldn't be doing it. I will create a new array from the plugs here to ensure it doesn't happen.
```suggestion
      .map((plugSet) => ({ ...plugSet, plugs: Array.from(plugSet.plugs).sort(sortPlugs) }))
```